satellite/satellitedb/nodeselection.go

```diff
@@ -271,7 +279,7 @@ func (partial partialQuery) asQuery() query {
 		fmt.Fprint(&q, " LIMIT ? ")
 		args = append(args, partial.limit)
 	} else {
-		fmt.Fprint(&q, ") filtered ORDER BY RANDOM() LIMIT ?")
+		fmt.Fprint(&q, ") filtered "+asOf+" ORDER BY RANDOM() LIMIT ?")
```

Comment:
Don't we need the same clause for the other branch?

Reply:
As discussed, the clause is not needed there.